samples/azure/storage-persistence/ASP_2/Client/Client.csproj

```diff
@@ -1,7 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
+  <PropertyGroup Condition=" '$(UsingMicrosoftNETSdk)' == 'true' ">
+    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(UsingMicrosoftNETSdk)' != 'true' ">
     <TargetFrameworks>net461</TargetFrameworks>
```

Comment:
Since this is a single framework, shouldn't these be `TargetFramework` instead of `TargetFrameworks`?